Api/Controllers/VisitsController.cs

```diff
@@ -1,12 +1,48 @@
+using Microsoft.AspNetCore.Identity;
 using Microsoft.AspNetCore.Mvc;
+using Reservant.Api.Models;
+using Reservant.Api.Models.Dtos.Visit;
 using Reservant.Api.Services;
+using Reservant.Api.Validation;
 
 namespace Reservant.Api.Controllers;
 
 /// <summary>
 /// Managing visits
 /// </summary>
 [ApiController, Route("/visits")]
-public class VisitsController(VisitService visitService) : Controller
+public class VisitsController(
+    VisitService visitService,
+    UserManager<User> userManager
+    ) : Controller
 {
+    
+    [HttpGet()]
+    [ProducesResponseType(200)]
+    [ProducesResponseType(400)]
+    public async Task<ActionResult<VisitSummaryVM>> GetVisits()
+    {
+        var user = await userManager.GetUserAsync(User);
+        var result = await visitService.GetVisitsAsync(user);
+
+        if (!result.IsError) return Ok(result.Value);
+
+        return result.ToValidationProblem();
+    }
```

Comment:
tego nie potrzebujemy bo michał t już to zrobił

Reply:
822820bc3b664b24d5582ec31926c6efdfdd547a